Shared/Samples/Augment reality to fly over scene/README.md

```diff
@@ -0,0 +1,40 @@
+# Augment reality to fly over scene
+
+Use augmented reality (AR) to quickly explore a scene more naturally than you could with a touch or mouse interface.
+
+![Image of augment reality to fly over scene](augment-reality-to-fly-over-scene.png)
+
+## Use case
+
+You can use AR to drop into an area and visualize information, like a proposed development or a historical model of a city. You could use flyover AR to explore a city by walking through it virtually.
+
+## How to use the sample
+
+When you open the sample, you'll be viewing the scene from above. You can walk around, using your device as a window into the scene. Try moving vertically to get closer to the ground.
+
+## How it works
+
+1. Create the `FlyoverSceneView` and add it to the view.
+2. Create the scene, add content, then display it.
+3. When the content you want to view loads, get its center point and use that to create the origin camera for the AR view. Note that the altitude should be set so that all scene content is visible. For a city, a good value might be a bit higher than the tallest building.
```

Comment:
"get its center point and use that to create the origin camera" the sample code doesn't do this. Have you tried getting the center of the mesh layer and using it as the initial location?

Reply:
Yes, but I didn't go with that implementation because there isn't a lot to look at in the center of the layer, and it can be kind of confusing when first opening the sample. The initial location is set to look at the church, like the [iOS implementation](https://github.com/Esri/arcgis-runtime-samples-ios/blob/main/arcgis-ios-sdk-samples/Augmented%20reality/Explore%20scenes%20in%20flyover%20AR/ExploreScenesInFlyoverAR.swift#L64-L65) did, which is why we added an `initialHeading` to the `FlyoverSceneView` in the toolkit. I left this in the readme, though, because I think that it is a good rule of thumb when using a flyover scene view for the first time. I can remove it if you think that is confusing.